prow/plugins.yaml

```diff
@@ -14,7 +14,8 @@ triggers:
 
 approve:
 - repos:
-  - kubernetes
+  - kubernetes/test-infra
+  - kubernetes/kubectl
   implicit_self_approve: true
```

Comment:
kubernetes/federation will default to `implicit_self_approve: false` and `issue_required: false`.